### PR TITLE
Tweak Veraison repo Readme to mention public meeting and reference Co…

### PR DESCRIPTION
**Pull request by SimonFrost-Arm**
…ntributing + Code of Conduct files (also added)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (thomas-fossati) [inline CODE_OF_CONDUCT.md:64]: we'll need to agree on something sensible to put here sooner or later, and update all the other instances accordingly :-)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Please make sure before you submit a pull request that a corresponding
+[Github issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-issues)
+exists where the problem you are trying to solve and any implementation approach can be discussed.
+
+Also, remember to
+[link](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
+your pull request to the corresponding issue.
+
+Please check the Contributing guidelines for any of the Veraison repos as they may add more guidance on appropriate Test or Documentation topics relevant to the contents of that repo.
```
Comment (thomas-fossati) [inline CONTRIBUTING.md:38]: nice

**Review by thomas-fossati — APPROVED** [review]
cool stuff!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (SimonFrost-Arm) [reply CODE_OF_CONDUCT.md:64]: I missed that bit! Seems that there should be a more private means of reporting but should we just add the Zulipchat site as a placeholder for now?

Comment (thomas-fossati) [reply CODE_OF_CONDUCT.md:64]: not sure a public channel is ideal... I'm opening an issue for this.

**Pull request merged**